tools/data_manipulation/xarray/xarray_coords_info.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="xarray_coords_info" name="NetCDF xarray Coordinate Info" version="0.18.2">
+    <description>Get values for each coordinate of a Netcdf file</description>
+    <requirements>
+        <requirement type="package" version="3">python</requirement>
+        <requirement type="package" version="1.5.6">netcdf4</requirement>
+        <requirement type="package" version="0.18.2">xarray</requirement>
+        <requirement type="package" version="0.9.0">geopandas</requirement>
+        <requirement type="package" version="1.7.1">shapely</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir output_dir &&
+        echo "Galaxy xarray version 0.18.2" > output_dir/version.tabular &&
+        python3 '$__tool_directory__/xarray_tool.py' '$input' --coords_info output_dir
+    ]]></command>
+    <inputs>
+        <param type="data" name="input" label="Netcdf file" format="netcdf,h5" help="Netcdf file you need to extract coordinate values."/>
+    </inputs>
+    <outputs>
+        <collection type="list" name="output_dir" label="Coordinates">
+            <discover_datasets pattern="__name_and_ext__" visible="true" directory="output_dir"/>
+        </collection>
+    </outputs>
+    <tests>
+        <test>
+             <param name="input" value="dataset-ibi-reanalysis-bio-005-003-monthly-regulargrid_1510914389133.nc"/>
+              <output_collection name="output_dir">
```

Comment:
```suggestion
              <output_collection name="output_dir" type="list" count="5">
```

Reply:
How big can such a collection be? One file per coordinate sounds dangerous to me :) Can this be 100.000 elements?

Reply:
no. it is related to the number of dimensions (latitude, longitude, depth, time, height, levels, etc.). So not so many (<20)